fix_prs.sh

```
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# Daily PR Fix Script
+# Fixes package-lock.json conflicts across all open PR branches
+
+cd pomofly
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)